C7/project.godot

```diff
@@ -31,6 +31,10 @@ window/size/height=768
 
 theme/custom="res://C7Theme.tres"
 
+[network]
+
+limits/debugger_stdout/max_chars_per_second=16384
```

Comment:
This lets the console print more stuff.  When I was printing out all the pictures we were importing, the console couldn't handle it until I added this.  Might be too much on really slow systems, but it seems like a generally good thing and didn't cause any problems on my 9.9-year-old system.